webpack/base.js

```diff
@@ -21,6 +21,10 @@ export default {
       '.jsx',
       '.svg',
     ],
+
+    alias: {
+      minim: path.resolve(__dirname, '../node_modules/minim/dist/minim.js'),
```

Comment:
The browser support for minim dist bundle is at https://github.com/refractproject/minim/blob/76949b53b472a1d4048803584b90f35f85ee0546/.babelrc#L5, just wanted to make sure this is similiar to what you support.

Reply:
Thanks for the link! I am actually after "not ie <11".

Would it be a problem to change it?

Reply:
Nope, feel free to test what you need and make a PR. If it's non breaking we can do a patch release.

Reply:
Created https://github.com/refractproject/minim/pull/226 which makes it the same as for phoenix-ui.